tests/test.js

```diff
@@ -1,6 +1,8 @@
 var Manager = require('../')
 var Dat = require('dat')
+var LocalDat = require('../dat')
```

Comment:
A bit awkward...

I wanted to test using the local one to make sure it worked on both ends too.